mobile_verifier/src/cli/server.rs

```diff
@@ -1,10 +1,11 @@
-use crate::{
-    verifier::{Verifier, VerifierDaemon},
-    Settings,
-};
+use crate::{verifier::VerifierDaemon, Settings};
```

Comment:
remind me, why is this module hosted under the cli dir ? 

nit: why is it called server.rs, i thought we previously moved away from this convention and named modules based on task/activity